### ISSUE TITLE
Test PSBT creation in GUI

### ISSUE THREAD

**Issue by Sjors** [bitcoin/bitcoin#17588]
#16944 added support for creating a [PSBT](https://github.com/bitcoin/bitcoin/blob/master/doc/psbt.md) in the GUI, when using a watch-only wallet.

It would be useful to test this functionally, by expanding the tests here:
https://github.com/bitcoin/bitcoin/blob/master/src/qt/test/wallettests.cpp#L173-L180

The test should also check the balance in the send screen, see #17587.

**Comment by brakmic** [comment]
@Sjors 

I've added a test-case for PSBT in my repo, but before opening a PR maybe we should check if it's acceptable at all.

https://github.com/brakmic/bitcoin/commit/b8f3e84f21f7bb9ef423210543ebe227820aef60

First, I've tried to create a transaction by using QT dialogs without much success. Therefore, I imported two *BuildTransaction*-Functions from other tests to create the needed transaction that could be used to initialize a PSBT. 

Regarding testing the **balance in send screen**: I still have to figure it out. :)
